.travis.yml

```diff
@@ -97,42 +107,63 @@ install:
 script:
 - set -e
 - if [[ "${BUILD_TARGET}" = "gcc_build" ]]; then
-    make BUILD_TYPE=Debug run_unit_tests -j4;
-    make clean;
-    make INSTALL_PREFIX=/usr BUILD_TYPE=Release run_unit_tests -j4;
+    cmake -DCMAKE_BUILD_TYPE=Debug -Bbuild/debug -H.;
+    make -Cbuild/debug -j4;
+    ./build/debug/src/unit_tests_runner;
+    cmake -DCMAKE_BUILD_TYPE=Release -Bbuild/release -H.;
+    make -Cbuild/release -j4;
```

Comment:
Is `-j4` correct for travis?

Reply:
No clue, I probably copied something I saw somewhere before. Like [here](https://github.com/Dronecode/DronecodeSDK/blob/develop/.travis.yml#L100).

Reply:
Oh I see :smile:

Reply:
Well, I don't find information on that, I'll just keep `-j4` for now.